src/lib.rs

```diff
@@ -288,12 +290,14 @@ impl Plugin for CosmicEditPlugin {
         app.add_systems(
             First,
             (
+                set_initial_scale,
                 cosmic_editor_builder,
                 placeholder_builder,
                 on_scale_factor_change,
                 render::cosmic_ui_to_canvas,
                 render::cosmic_sprite_to_canvas,
-            ),
+            )
+                .chain(),
```

Comment:
hmmm, do we really need `chain`?

Maybe it's better to add `after(set_initial_scale)` to every system in First schedule?

Reply:
yeah i was having trouble remembering how scheduling works now, still got the old way in my head and used the first thing that worked in my LSP >.<